README.md

```diff
@@ -11,19 +15,24 @@ Here is how you go about doing that.
     "preinstall": "make deploy-key-setup || true",
     "postinstall": "make deploy-key-teardown || true"
   }
+
+"dependencies": {
+  ...
+  "shyp-toolkit": "git://github.com/Shyp/shyp-toolkit",
```

Comment:
two small thoughts:
- should probably lock this down to a specific version
- npm has a [shorter syntax](https://docs.npmjs.com/files/package.json#github-urls)

so, this could look like:
```
 "shyp-toolkit": "Shyp/shyp-toolkit#v1.0.0",
```